lib/vrl/tests/src/test.rs

```diff
@@ -0,0 +1,141 @@
+use std::collections::BTreeMap;
```

Comment:
This is mostly copy/pasted from `main.rs` to do some clean-up, since we now also have a `docs.rs` to parse cue docs as tests.